js/randomUser.js

```diff
@@ -0,0 +1,119 @@
+import utils from './utils.js';
+
+const descriptions = [
+  'With ten years of experience in [genitive] backpack [name] comes to our course to teach the art (as [gender] calls it) of programming in [language]. As [gender] believes that life is about giving and teaching is giving, [gender] will give you whatever you will need to be the best.',
+  'Author of the best selling books about [language], [name] joined our board of teachers in order to spread [genitive] knowledge with our students. You will meet [dative] in our classes every Monday and Thursday.',
+  'There is nothing better than [language]. Or so [name] says. Sometimes [gender] will allow [dative]self to walk through our virtual campus and read some sketchy literature, but rest assured: [gender] will connect with people just like [gender] connects with book characters.',
+  'Sometimes life gives you lemons. And sometimes it gives you [language]. Sometimes it gives you lemons and [language] and [name]. And then is the time to start making some lemonade with [genitive] help. Prepare yourself for this course and prepare a good lemonade too!',
+  'Three times winner of the best designer award, [name] changed [genitive] careere more than once ([genitive] political views too). When [gender] started programming [gender] had no idea of the vast universe that will open up in front of [dative]. Now [gender] is in love with programming and so will you!',
+  'For the past 24 years [name] was the head of the [language] team for Google. While [gender] is not restricting [dative]self to [language], [gender] keeps at heart this programming language, so [gender] secretly wants everyone to learn it.',
+];
+
+async function populateUsers(speakers) {
+  const container = utils.qs('.speakers-container');
+
+  speakers.forEach((speaker) => {
+    const language = ['JavaScript', 'PHP', 'C#', 'Python', 'Ruby', 'Kotlin'][
+      Math.floor(Math.random() * 6)
+    ];
+
+    const article = utils.createElement({
+      tagName: 'article',
+    });
```

Comment:
Well done 🤯 Great use of propper tags to have good semantics when generating the dynamic data 🤓